app/controllers/users_controller.rb

```diff
@@ -15,4 +15,8 @@ def update
       render :edit
     end
   end
+
+  def bids_placed
```

Comment:
I think the standardized naming conventions are really helpful from a developer perspective.